<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>usepage</class>
 <widget class="QWidget" name="usepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>581</width>
     <height>391</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>631</width>
     <height>391</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>121</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>在售商品数</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="sellgoods_number_lcdNumber">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>今日销售量</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="sell_num_lcdNumber">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>今日销售额</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="money_lcdNumber">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>今日销售最佳商品</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_9">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>7</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>销售量：</string>
          </property>
         </widget>
         <widget class="QLabel" name="bestgoods_sellnumber_label">
          <property name="geometry">
           <rect>
            <x>47</x>
            <y>7</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_11">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>7</x>
            <y>7</y>
            <width>36</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>销售额：</string>
          </property>
         </widget>
         <widget class="QLabel" name="bestgoods_money_label">
          <property name="geometry">
           <rect>
            <x>47</x>
            <y>7</y>
            <width>101</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>商品名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="bestgoods_name_label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>400</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>400</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>101</width>
      <height>371</height>
     </size>
    </property>
    <property name="title">
     <string>工具栏</string>
    </property>
    <widget class="QToolBox" name="analyse_page">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>341</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="goods_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>79</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="label">
       <string>管理商品信息</string>
      </attribute>
      <widget class="register_lable" name="add_goods_label">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>0</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>添加商品</string>
       </property>
      </widget>
      <widget class="register_lable" name="del_goods_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>删除商品</string>
       </property>
      </widget>
      <widget class="register_lable" name="change_goods_label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>更改商品信息</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="inventory_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>79</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="label">
       <string>库存管理</string>
      </attribute>
      <widget class="register_lable" name="view_inventory_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>查看库存</string>
       </property>
      </widget>
      <widget class="register_lable" name="edit_inventory_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>编辑库存</string>
       </property>
      </widget>
      <widget class="register_lable" name="report_inventory_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>库存报告</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="analyse_page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>79</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="label">
       <string>商品分析</string>
      </attribute>
      <widget class="register_lable" name="profit_label">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>0</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>商品毛利</string>
       </property>
      </widget>
      <widget class="register_lable" name="ranking_label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>20</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>销售排名（周）</string>
       </property>
      </widget>
      <widget class="register_lable" name="unsalabel_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>滞销商品</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="data_page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>79</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="label">
       <string>数据</string>
      </attribute>
      <widget class="register_lable" name="datain_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>数据导入</string>
       </property>
      </widget>
      <widget class="register_lable" name="dataout_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>40</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>数据导出</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>361</width>
       <height>45</height>
      </rect>
     </property>
     <property name="title">
      <string>用户</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>348</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="printuser_label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="register_lable" name="label">
        <property name="text">
         <string>登出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>register_lable</class>
   <extends>QLabel</extends>
   <header>register_lable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
